docs/src/user-guide/getting-started.md

```diff
@@ -28,14 +28,15 @@ If you want to use a specific shareable config that is hosted on npm, you can us
 
 ```shell
 # use `eslint-config-semistandard` shared config
-# npm 6.x
-npm init @eslint/config --config semistandard
-
 # ⚠️ npm 7+, extra double-dash is needed:
```

Comment:
If the `npm 7+` example will be first, it might make sense to move ⚠️ to the `npm 6.x` example and say there that there should be no double-dash.

Reply:
changed in [702a1a8](https://github.com/eslint/eslint/pull/16596/commits/702a1a8d465c290a6562a057f42c85c6526f80f2)